tests/containers/base_image_test.py

```diff
@@ -104,18 +106,7 @@ def check_elf_file():
                 with subtests.test(f"{dlib=}"):
                     pytest.fail(f"{dlib=} has unsatisfied dependencies {deps=}")
 
-    def test_oc_command_runs(self, image: str):
```

Comment:
am I missing something or this test was just removed without any replacement? :thinking:

Reply:
you're right, my bad

Reply:
okay, you resolved this, but I must be blind as I still don't see it anywhere? :thinking:

Reply:
i did not resolve it before turns out, resolved it now

it was still missing, you'r not blind